implementations/rust/ockam/ockam_api/src/cli_state/storage/enrollments_repository.rs

```diff
@@ -0,0 +1,30 @@
+use crate::cli_state::enrollments::IdentityEnrollment;
+use ockam::identity::Identifier;
+use ockam_core::async_trait;
+use ockam_core::Result;
+
+/// This trait stores the enrollment status for identities
+/// If an identity has been enrolled it is possible to retrieve:
+///
+///  - its name (if it has one)
+///  - if this the default identity
+///  - if an identity was enrolled and when the local node was informed
+///
+///
+#[async_trait]
+pub trait EnrollmentsRepository: Send + Sync + 'static {
```

Comment:
This is mostly for storing a flag that we keep locally to know if an identifier has been enrolled on the default project. In a way this can also be seen as a cache for something that happens on a remote authority. Please suggest something if you have a better name.

Reply:
I think just adding `local` to this sentence:

`This trait stores the enrollment status for local identities`

Reply:
Done.